apps/clientApp/src/androidMain/kotlin/network/bisq/mobile/client/common/domain/access/security/SanVerifier.kt

```
@@ -0,0 +1,36 @@
+                        value as String?,
+                        ignoreCase = true,
+                    )
+                ) {
+                    return true
+                }
+                if (type == 7 && host == value) {
+                    return true
+                }
+            }
+            return false
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)